docker/Dockerfile

```diff
@@ -1,8 +1,9 @@
-# TODO: Move this file into packages/radio
```

Comment:
Retain this

Reply:
done here b9d17a524cc8c4576c43a2b9fb8d787412d13e62